### ISSUE TITLE
Get involved

### ISSUE THREAD

**Issue by yavko** [PolyMC/polymc.github.io#78]
IMO the get involved part needs to be remade, I'm happy to do so and will create a pr, just wanna create this issue as a tracker of sorts.

**Comment by ZekeSmith** [comment]
Yeah that's fair, it could probably also be put on another page the community thing links to it already. I might be able to make it in my own time but if you have the time and can make a pr that would be great.

**Comment by yavko** [comment]
> ve the time and can make a pr that would be great.

Ok I'll make it ASAP!

**Comment by yavko** [comment]
Okay I've started it, and have a rough idea on how it'll look I'll report once its done.

**Comment by ZekeSmith** [comment]
Sounds good

**Comment by yavko** [comment]
@ZekeSmith I'm adding icons for the respective brands, but should I have them as pngs or svgs, I think using vector graphics would be better but the rest of the website uses raster images so idk what to chose thx.

**Comment by ZekeSmith** [comment]
I didnt get this message sorry but if your still working on this, icons use svgs.

**Comment by ZekeSmith** [comment]
Any information regarding your progress @yavko I am happy to finish the rest of the work.

**Comment by yavko** [comment]
> Any information regarding your progress @yavko I am happy to finish the rest of the work.

I have most of it done I just need to tweak it a bit, sorry for not finishing earlier at the time I had exams and it took over, but I'll try to get it in tomorrow or maybe even today.

**Comment by DioEgizio** [comment]
""tomorrow""

![haha funny](https://user-images.githubusercontent.com/83089242/180664909-e7dc0b3a-182e-4d05-ba9a-3c7e3532d4f8.png)

**Comment by yavko** [comment]
> ""tomorrow""
> 
> ![haha funny](https://user-images.githubusercontent.com/83089242/180664909-e7dc0b3a-182e-4d05-ba9a-3c7e3532d4f8.png)

Yeah I'm really bad at doing one thing at a time

**Comment by yavko** [comment]
Anyways I opened the pr, and added some new changes, I'm also adding a wiki page for extra details

**Comment by yavko** [comment]
Okay wiki page for matrix complete, I also made the icons actually clickable

**Comment by yavko** [comment]
All I need to do is finishing touches for the layout as said in the pr todo
